app/views/curation_concerns/file_sets/media_display/_default.html.erb

```diff
@@ -1,3 +1,6 @@
 <div class="no-preview">
   <%= t('sufia.works.show.no_preview') %>
+  <% if CurationConcerns.config.display_media_download_link %>
+    <p/><%= link_to t('sufia.file_set.show.download'), main_app.download_path(file_set), target: "_new", title: "Download the document" %>
```

Comment:
I'll do the same in CC with "file" versus "document." Document was already there, so I just kept it.

https://github.com/projecthydra/curation_concerns/pull/927/files#diff-ceb1b6efe986c3ac749164a232c1e8e5R21